Src/Base/AMReX_MFIter.H

```diff
@@ -11,7 +11,7 @@
 
 #include <AMReX_Gpu.H>
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
```

Comment:
Just to make usage in complex user code easier, we might want to write
```suggestion
#if defined(AMREX_USE_OMP) || defined(_OPENMP)
```
?

Reply:
`AMReX_Config.H` contains
```
#if defined(AMREX_USE_OMP) && !defined(_OPENMP)                                                                                                                                                            
#error "libamrex was built with OpenMP, so the downstream project must activate it, too"                                                                                                                   
#endif 
```
So we do not allow the downstream project to build without OpenMP if AMReX is built with OpenMP.